subprojects/docs/src/docs/release/notes.md

```diff
@@ -165,9 +167,17 @@ networkTimeout=30000
 
 See the [user manual](userguide/gradle_wrapper.html#sec:adding_wrapper) for more information.
 
+### Command-line improvements
+
+#### Tasks can be requested to re-run selectively 
+
+A new implicit `--rerun` option is now available for every task. The effect is similar to `--rerun-tasks`, but it forces a rerun only on the specific task to which it was directly applied. 
```

Comment:
I'd suggest promptly mentioning that one needs to specify the `--rerun` option after the task path, so that, for users who have the context, it's clear how to try it without looking through the docs.